app/helpers/filters_helper.rb

```diff
@@ -93,7 +93,11 @@ def assigned_to_filter_options(user)
   end
 
   def collection_year_options
-    { "2023": "2023/24", "2022": "2022/23", "2021": "2021/22" }
+    {
+      current_collection_start_year.to_s => year_combo(current_collection_start_year),
+      previous_collection_start_year.to_s => year_combo(previous_collection_start_year),
+      twice_previous_collection_start_year.to_s => year_combo(twice_previous_collection_start_year),
+    }
```

Comment:
There's `years_of_available_lettings_forms` and `years_of_available_sales_forms` in the `FormHandler` which would only return the years for the forms we currently hold (that does include `next` form as well, so not sure if it would be useful to reuse that)

Reply:
Also, not sure if helpful, but I've been calling the form before the previous `archived` in the rest of the codebase

Reply:
Ah that could be useful, I'll have a look, thanks!

Reply:
Think I'll leave as is to be explicit about the three years we want here (as it's not even the standard previous, current, next we used to have elsewhere) but I'll rename to `archived`